app/components/layout/nav.tsx

```diff
@@ -86,26 +109,27 @@ export function Nav() {
             })}
           </div>
           <div className="pr-18 hidden items-center justify-end text-lg lg:mr-6 lg:flex lg:w-[15%] lg:space-x-8 lg:text-xs xl:mr-14 xl:w-[10%]">
-            <button className="xl:text-md h-10 w-40 cursor-not-allowed rounded-full bg-cyan-500 text-xs font-bold text-white lg:h-8 lg:w-24 xl:h-10 xl:w-40 2xl:h-10 2xl:w-40 2xl:text-[16px]">
-              Sign Up (Soon)
-            </button>
+          <Button className="bg-cyan-500 text-xs h-10 hover:bg-cyan-500 cursor-not-allowed">
```

Comment:
do not override the color, because Button component is using primary color, which is the same color as Figma design